config/routes.rb

```diff
@@ -16,5 +16,10 @@
     resources :items, only: [:index]
   end
 
+  namespace :admin do
+    get '/dashboard', to: 'base#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.